packages/vue/src/components/atoms/SfCircleIcon/SfCircleIcon.html

```diff
@@ -1,6 +1,12 @@
-<SfButton class="sf-circle-icon" v-on="$listeners">
+<SfButton class="sf-circle-icon" v-on="$listeners" :aria-label="alt || icon">
```

Comment:
`aria-label` should never be the value of `icon` as `icon` can be SVG path, which will make things bad. Since we make `alt` required it should be enough. Don't need a fallback.

Reply:
https://github.com/DivanteLtd/storefront-ui/pull/492/commits/ee9b7dddad2bb90a1f59a2827b89ad74317377d5